iai_apartment/urdf/kitchen.xarco

```diff
@@ -1,38 +1,35 @@
-<?xml version="1.0"?>
-
-<!-- Such is life in a virtual world -->
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="kitchen">
 
-	<!-- Such is life in a kitchen : xacro -->
+	
 	<xacro:macro name="kitchen_macro" params="name parent *origin"> 
 
 		<joint name="${name}" type="fixed">
 			<xacro:insert_block name="origin" />
 			<parent link="${parent}" />
-			<child link="kitchen_root"/>
+			<child link="kitchen_root" />
 		</joint>
 		<link name="kitchen_root" />
 
-<!-- Frames, reference frames -->
-	<joint name="frame_A_joint" type="fixed">
-		<parent link="kitchen_root" />
-		<origin xyz="0.5 0.5 0.080" />
-		<child link="side_A"/>
-	</joint>
 
-	<!-- Bond, James Bond -->
-	<joint name="frame_B_joint" type="fixed">
-		<parent link="kitchen_root" />
-		<origin xyz="3 3.5 0.092" rpy="0 0 3.14"/>
-		<child link="side_B"/>
-	</joint>
+		<joint name="frame_A_joint" type="fixed">
+			<parent link="kitchen_root" />
+			<origin xyz="0.5 0.5 0.080" />
+			<child link="side_A" />
+		</joint>
+
+		
+		<joint name="frame_B_joint" type="fixed">
+			<parent link="kitchen_root" />
+			<origin xyz="3 3.5 0.092" rpy="0 0 3.14" />
+			<child link="side_B" />
+		</joint>
+
+		<link name="side_A" />
 
-	<link name="side_A" />
+		<link name="side_B" /> 
 
-	<link name="side_B" /> 
 
-<!-- Cabinets side A -->
```

Comment:
This comment serves a purpose. This file is currently (new version) about 1750 lines long. Either this should turn into a hell of nested xacro files, or these comments serve as folding points. Without them, you can't easily fold and compare the snippets of code. I suggest you add something to your script to ignore the comments altogether. if you want to change individual comments because you think they're not good or don't explain enough, by all means, go ahead. but removing the structural ones, just makes any further maintenance a pain.